.github/workflows/lints.yml

```diff
@@ -20,7 +20,6 @@ env:
 jobs:
   lint:
```

Comment:
I know there was a comment about this but I'm not sure I agree, I think this should not be ran for drafts